src/app/clusters/scenes-server/SceneTableImpl.cpp

```diff
@@ -19,686 +19,195 @@
-        {
-            // If Find doesn't return CHIP_NO_ERROR, the scene wasn't found, which doesn't return an error
-            VerifyOrReturnValue(this->Find(scene_id, scene.index) == CHIP_NO_ERROR, CHIP_NO_ERROR);
-
-            // Update the global scene count
-            EndpointSceneCount endpoint_scene_count(endpoint_id);
-            ReturnErrorOnFailure(endpoint_scene_count.Load(storage));
-            endpoint_scene_count.count_value--;
-            ReturnErrorOnFailure(endpoint_scene_count.Save(storage));
-
-            scene_count--;
-            scene_map[scene.index].Clear();
-            err = this->Save(storage);
-
-            // On failure to update the scene map, undo the global count modification
-            if (CHIP_NO_ERROR != err)
-            {
-                endpoint_scene_count.count_value++;
-                ReturnErrorOnFailure(endpoint_scene_count.Save(storage));
-                return err;
-            }
-
-            err = scene.Delete(storage);
-
-            // On failure to delete scene, undo the change to the Fabric Scene Data and the global scene count
-            if (CHIP_NO_ERROR != err)
-            {
-                endpoint_scene_count.count_value++;
-                ReturnErrorOnFailure(endpoint_scene_count.Save(storage));
-
-                scene_count++;
-                scene_map[scene.index] = scene.mStorageId;
-                ReturnErrorOnFailure(this->Save(storage));
-                return err;
-            }
-        }
-        return err;
-    }
-
-    CHIP_ERROR Load(PersistentStorageDelegate * storage) override
-    {
-        VerifyOrReturnError(nullptr != storage, CHIP_ERROR_INVALID_ARGUMENT);
-        uint8_t deleted_scenes_count = 0;
-
-        uint8_t buffer[kPersistentFabricBufferMax] = { 0 };
-        StorageKeyName key                         = StorageKeyName::Uninitialized();
-
-        // Set data to defaults
-        Clear();
-
-        // Update storage key
-        ReturnErrorOnFailure(UpdateKey(key));
+    return 128;
+}
 
-        // Load the serialized data
-        uint16_t size  = static_cast<uint16_t>(sizeof(buffer));
-        CHIP_ERROR err = storage->SyncGetKeyValue(key.KeyName(), buffer, size);
-        VerifyOrReturnError(CHIP_ERROR_PERSISTED_STORAGE_VALUE_NOT_FOUND != err, CHIP_ERROR_NOT_FOUND);
-        ReturnErrorOnFailure(err);
+template <>
+constexpr EndpointId DefaultSceneTableImpl::Super::kEntryEndpointClusterID()
+{
+    return chip::app::Clusters::ScenesManagement::Id;
+}
 
-        // Decode serialized data
-        TLV::TLVReader reader;
-        reader.Init(buffer, size);
+#include <app/storage/FabricTableImpl.hpp>
 
-        err = Deserialize(reader, storage, deleted_scenes_count);
+namespace {
+/// @brief Tags Used to serialize Scenes so they can be stored in flash memory.
+/// kGroupID: Tag for GroupID if the Scene is a Group Scene
+/// kSceneID: Tag for the scene ID together with the two previous tag, forms the SceneStorageID
```

Comment:
Which "two previous tag"?

Reply:
Updated (it's just kGroupID; not sure why it says two previous)